README.md

```diff
@@ -263,3 +263,15 @@ To obtain the dependencies, `cd` into the atrament directory and run `npm instal
 You should be able to then build atrament by simply running `npm run build`.
 
 I didn't bother writing tests because it's such a small package. Contributions are welcome!
+
+### Run a local server
+
+First build the project:
+```sh
+npm run build
+```
+
+Then start a local server:
+```sh
+npm start
+```
```

Comment:
thanks! I don't think every library needs its own dev server setup, it's usually relatively pain-free to start a static HTML server in the `demo` directory and just have `npm run watch` refreshing the bundle. But I guess it's worth making it a little simpler for Atrament contributors.